cdap-docs/examples-manual/source/examples/purchase.rst

```diff
@@ -26,28 +26,28 @@ and write to an ObjectStore Dataset.
   - The ``PurchaseFlow`` reads the ``purchaseStream`` and converts every input String into a
     Purchase object and stores the object in the *purchases* Dataset.
   - User profile information for the user can be added by using ``curl`` calls (or another method) which are
-    then stored in the *userProfiles* Dataset.
-  - The ``CatalogLookupService`` fetches the catalog id for a given product. The CatalogLookupService
-    is called from the PurchaseStore Flowlet. The host and port of the CatalogLookupService is discovered
+    then stored in the ``userProfiles`` Dataset.
+  - The ``CatalogLookup`` Service fetches the catalog id for a given product. The ``CatalogLookup`` Service
+    is called from the PurchaseStore Flowlet. The host and port of the ``CatalogLookup`` Service is discovered
     using the Service discovery framework.
   - The ``UserProfileService`` is responsible for storing and retrieving the user information
-    for a given user id from the *userProfiles* Dataset. The host and port of the ``UserProfileService`` is
+    for a given user id from the ``userProfiles`` Dataset. The host and port of the ``UserProfileService`` is
     discovered using the Service discovery framework.
   - When scheduled by the ``PurchaseHistoryWorkFlow``, the ``PurchaseHistoryBuilder`` MapReduce
-    reads the *purchases* Dataset. It fetches the user profile information, if it is available, from
+    reads the ``purchases`` Dataset. It fetches the user profile information, if it is available, from
     the ``UserProfileService`` and creates a purchase history. It stores the purchase history in the
-    *history* Dataset every morning at 4:00 A.M. using a Time Schedule, and also every time 1MB of data
+    ``history`` Dataset every morning at 4:00 A.M. using a Time Schedule, and also every time 1MB of data
     is ingested by the ``purchaseStream`` using a Data Schedule.
   - You can either manually (in the Process screen of the CDAP Console) or 
     programmatically execute the ``PurchaseHistoryBuilder`` MapReduce to store 
-    customers' purchase history in the *history* Dataset.
-  - Request the ``PurchaseHistoryService`` retrieve from the *history* Dataset the purchase history of a user.
-  - Execute a SQL query over the *history* Dataset. You can do this using a series of ``curl``
+    customers' purchase history in the ``history`` Dataset.
+  - Request the ``PurchaseHistoryService`` retrieve from the ``history`` Dataset the purchase history of a user.
```

Comment:
This sentence seems malformed/has a typo - perhaps a missing "to" before retrieve?